ddtrace/contrib/dbapi/__init__.py

```diff
@@ -155,23 +154,16 @@ def fetchmany(self, *args, **kwargs):
                                   *args, **kwargs)
 
 
-def _get_config(new_cfg):
-    # Need to backwards support the dbapi2 config entry
-    # but give precedence to the given config.
-    cfg = config.dbapi2.copy()
-    cfg.update(new_cfg)
-    return cfg
-
-
 class TracedConnection(wrapt.ObjectProxy):
     """ TracedConnection wraps a Connection with tracing code. """
 
     def __init__(self, conn, pin=None, cfg=None, cursor_cls=None):
+        cfg = cfg or config.dbapi2
```

Comment:
Seems like we can/should do away with dbapi2 altogether?

Reply:
Deprecating it and we can remove it afterwards.